README.md

```diff
@@ -1,10 +1,10 @@
 # Flow
 
-_"Name-giving will be the foundation of our science."_ - Linnaeus
+_"Name-giving will be the foundation of our science."_ &mdash;Linnaeus
```

Comment:
Should be pretentious, yes, but should also have consistent spacing. There should either be no leading space before the dasy, or there should be a trailing after it. I prefer the latter.